### ISSUE TITLE
Time series is broken on v2.0.1 when query has no metric name column

### ISSUE THREAD

**Issue by gilkgit** [grafana/azure-data-explorer-datasource#60]
Time series is broken on v2.0.1 when query has no metric name column.
for example
This works:
print now(), 5, name ="a"

but this fails :
print now(), 5

Error displayed to user:
SyntaxError: Unexpected token c in JSON at position 0

(installed the new plugin both on windows and Linux Grafana deployments)

**Comment by gilkgit** [comment]
I'm not sure I explained well the scenario and it's importance.
Charts of type : sometable | summarize count() by bin(Time,1d)
are now broken. Taking into consideration the other bug I opened,[ where variables have stopped working](https://github.com/grafana/azure-data-explorer-datasource/issues/61), I fear that ver 2.0.1 is quite broken.

**Comment by matthiasdg** [comment]
I have the same issue; e.g. 
```
events
| where $__timeFilter(timestamp) | summarize count()  by bin(timestamp, 1h) | order by timestamp desc
```
also results in the unexpected token c -> if I look at the query inspector I think a json object is expected instead of a string here
```
 "series": [
          {
            "name": "count_",
            "points": [
              [
```
(if I use an alias for the count, the unexpected token changes)

**Comment by wollollo** [comment]
Same issue. A query of type
`table | summarize avg(value) by bin(timestamp, 1h) | order by timestamp asc`
fails for time series. (Setting 'format as' to table works.) Adding in a
`project name="nonsense", timestamp, avg_value` is a workaround.

**Comment by srclosson** [comment]
Found a bug and fixed. Makes sense, was with the aliasing changes made for 2.0.1. If the backend did not return valid JSON, alias parsing failed. Create the proper structure now so any query that does not look like json get parsed properly.

**Comment by briangann** [comment]
fixed in release 2.0.2 #68

**Comment by gilkgit** [comment]
Great!
I wonder if that is also related to #61 which happened around the same time.
